_data/patcher-landing-page-tabs-terraform-upgrade.yml

```diff
@@ -0,0 +1,7 @@
+- id: pr
+  title: PR
```

Comment:
Perhaps "Example Pull Request" would be a better title?

Reply:
Done in 8a1c45c.